.github/workflows/fetch-icons.yaml

```diff
@@ -0,0 +1,112 @@
+        with:
+          fetch-depth: 0
+
+      - name: Setup
+        uses: swisspost/design-system/.github/actions/setup-pnpm@main
+
+      - name: Get Date
+        id: current-date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Create Branch
+        id: current-branch
+        run: |
+          git checkout -b $BRANCH_NAME
+          echo "::set-output name=branch::$BRANCH_NAME"
+        env:
+          BRANCH_NAME: ${{ steps.current-date.outputs.date }}-update-icon-svgs
+
+      - name: Fetch Icons
+        run: pnpm --filter design-system-icons fetchSVGs
+
+      - name: Get Changes
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          files: ./packages/icons/public/svg/**
+
+      - name: Exit if no changes
+        if: steps.changed-files.outputs.any_modified == 'false'
+        run: |
+          git checkout main
+          git branch --delete ${{ steps.current-branch.outputs.branch }}
+          exit 1
+
+      - name: Create Changeset
+        run: |
+          echo ---                                      > $CHANGESET
+          echo '@swisspost/design-system-icons': patch >> $CHANGESET
+          echo ---                                     >> $CHANGESET
+          echo                                         >> $CHANGESET
+          echo Updated icon SVGs.                      >> $CHANGESET
+          echo                                         >> $CHANGESET
+        env:
+          CHANGESET: ./.changeset/${{ steps.current-branch.outputs.branch }}.md
+
+      - name: Commit Changes
+        run: |
+          git config --global user.name github-actions[bot]
+          git config --global user.email github-actions@github.com
+          git add .
+          git commit -am "chore(icons): :point_up: update icon SVGs"
+          git push -u origin ${{ steps.current-branch.outputs.branch }}
+
+      - name: Write PR Body
+        id: pr-body
+        run: |
+          echo "# Design system icons are now up to date!"                                              > $PR_BODY_FILE
+          echo "::set-output name=file::$PR_BODY_FILE"
+
+          UPDATED_ICONS=($MODIFIED_ICONS $RENAMED_ICONS $COPIED_ICONS)
+
+          declare -a labels;            declare -A changes
+          labels+=( "Added icons" );    changes["Added icons"]=$ADDED_ICONS
+          labels+=( "Updated icons" );  changes["Updated icons"]=$UPDATED_ICONS
+          labels+=( "Deleted icons" );  changes["Deleted icons"]=$ADDED_ICONS
+
+          for label in "${labels[@]}"; do
+            if [[ ${changes[$label]} != '' ]]; then
+              echo "## $label"                                                                         >> $PR_BODY_FILE
+              for file in ${changes[$label]}; do
+                echo "- $(echo $file | sed 's/^packages\/icons\/public\/svg\///' | sed 's/\.svg$//')"  >> $PR_BODY_FILE
+              done
+            fi
+          done
```

Comment:
I think this needs some explanation as well. Is this just writing the PR body file or setting git labels as well?

Reply:
This is currently only for the PR body. Do I also need to add tags?

Reply:
No, git tags would be unnecessary. In this case, everything is good.